crates/bevy_pbr/src/render/skin.rs

```diff
@@ -188,6 +184,28 @@ impl SkinUniformInfo {
     }
 }
 
+#[derive(Resource, Clone, Debug, ExtractResource, Reflect)]
+#[reflect(Resource, Default, Debug, Clone)]
+pub struct GlobalSkinnedMeshSettings {
```

Comment:
My instinct is that it will be clearer and more efficient if we simply split this into two resources. They can live beside each other and be crosslinked though.